torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9974,6 +9989,14 @@ def ref_pairwise_distance(input1, input2):
            dtypes=floating_and_complex_types(),
            sample_inputs_func=sample_inputs_linalg_slogdet,
            decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack],),
+    OpInfo('linalg.trace',
```

Comment:
It'd be good to add a line to the OpInfo:
```python
ref=np.trace,
```
this way we would check that the implementation is correct, comparing it against numpy.